gruntfileConfig.json

```diff
@@ -3,7 +3,7 @@
   "build": "./html",
   "src": "./src",
   "assets": "./src/assets",
-  "theme": "./node_modules/patternpack-example-theme",
+  "theme": "patternpack-example-theme",
```

Comment:
Updated to demonstrate the simplified theme configuration using only the package name.